### ISSUE TITLE
No `repeat` function in Version 2.3.4

### ISSUE THREAD

**Issue by daniel-vera-g** [tiaanduplessis/pomd#34]
## Expected Behavior

As in the README specified:

`start --time 20:00 --chill 03:00 --repeat 6` or `start -t 20m -c 3m -r 6` should work.

## Current Behavior

`Invalid option: 'r'...` and `Invalid option: 'repeat'...`

## Steps to Reproduce

As in the README:

1. `npm install -g pomd`
2. `pomd`
3. `help start` -> NO `repeat`option:

```
 Usage: start [options]

  Start a Pomodoro

  Options:

    --help               output usage information
    -t, --time <time>    Set the time of the Pomodoro. Default is 25:00 minutes.
    -c, --chill <chill>  Set the time of chill. Default is 5:00 minutes.
    -l, --loop           Run continuous Pomodoros.
```

4. `start --time 20:00 --chill 03:00 --repeat 6` and `start -t 20m -c 3m -r 6` see error output above.

## Context (Environment)

1. Node: v8.16.0
2. npm: 6.4.1
3. pomd: pomd@2.3.4
4. OS: Ubuntu 18.04

**Comment by daniel-vera-g** [comment]
@tiaanduplessis After checking the source code for v2.3.4 and https://www.npmjs.com/package/pomd description, it seeems that v2.3.4 does not have the `repeat` feature. Would you mind creating a new release with the new code to update the package?

**Comment by tiaanduplessis** [comment]
@daniel-vera-g good spot. Just released `v2.4.5`

**Comment by daniel-vera-g** [comment]
> @daniel-vera-g good spot. Just released `v2.4.5`

Thx, now it works ;)
